barretenberg/cpp/src/barretenberg/honk/proof_system/permutation_library.hpp

```diff
@@ -7,166 +7,6 @@
 
 namespace bb {
 
-/**
```

Comment:
These two methods were just duplicates of equivalent ones in grand_product_library (with slightly different names).

Reply:
How have you verified these are equivalent?

Reply:
Confirmed that these were literally identical code aside from a couple of trivial syntactic differences